src/main/java/org/fundacionjala/sfdc/pageobjects/opportunities/SFONewModifyPage.java

```diff
@@ -12,61 +12,281 @@
+
     /**
      * @param closeDate setOpportunityCloseDate.
      */
     private void setOpportunityCloseDate(final String closeDate) {
         CommonActions.setTextElement(this.opportunityCloseDate, closeDate);
     }
 
+    /**
+     * @param opportunityNextStep setOpportunityNextStep.
+     */
+    private void setOpportunityNextStep(final String opportunityNextStep) {
+        CommonActions.setTextElement(this.opportunityNextStep, opportunityNextStep);
+    }
+
     /**
      * @param stage setOpportunityStage.
      */
     private void setOpportunityStage(final String stage) {
         CommonActions.setTextElement(this.opportunityStage, stage);
     }
 
+    /**
+     * @param opportunityProbability setOpportunityProbability.
+     */
+    private void setOpportunityProbability(final String opportunityProbability) {
+        CommonActions.setTextElement(this.opportunityProbability, opportunityProbability);
+    }
+
+    /**
+     * @param primaryCampaignSource setPrimaryCampaignSource.
+     */
+    private void setPrimaryCampaignSource(final String primaryCampaignSource) {
+        CommonActions.setTextElement(this.primaryCampaignSource, primaryCampaignSource);
+    }
+
+    /**
+     * @param orderNumber setOrderNumber.
+     */
+    private void setOrderNumber(final String orderNumber) {
+        CommonActions.setTextElement(this.orderNumber, orderNumber);
+    }
+
+    /**
+     * @param currentGenerator setCurrentGenerator.
+     */
+    private void setCurrentGenerator(final String currentGenerator) {
+        CommonActions.setTextElement(this.currentGenerator, currentGenerator);
+    }
+
+    /**
+     * @param trackingNumber setTrackingNumber.
+     */
+    private void setTrackingNumber(final String trackingNumber) {
+        CommonActions.setTextElement(this.trackingNumber, trackingNumber);
+    }
+
+    /**
+     * @param mainCompetitors setMainCompetitors.
+     */
+    private void setMainCompetitors(final String mainCompetitors) {
+        CommonActions.setTextElement(this.mainCompetitors, mainCompetitors);
+    }
+
+    /**
+     * @param deliveryInstallationStatus setDeliveryInstallationStatus.
+     */
+    private void setDeliveryInstallationStatus(final String deliveryInstallationStatus) {
+        CommonActions.selectOnComboBox(this.deliveryInstallationStatus, deliveryInstallationStatus);
+    }
+
+    /**
+     * @param description setDescription.
+     */
+    private void setDescription(final String description) {
+        CommonActions.setTextElement(this.description, description);
+    }
 
     /**
-     * @param values contact field
-     * @return map of contact fields.
+     * @param values opportunity field
+     * @return map of opportunity fields.
      */
     public Map<OpportInputs, Value> getStrategyStepMap(final Map<OpportInputs, String> values) {
         EnumMap<OpportInputs, Value> strategyMap = new EnumMap<>(OpportInputs.class);
 
+        strategyMap.put(OpportInputs.OPPORTUNITY_PRIVATE, () ->
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)